plinic/src/components/input/Input.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+
+function Input({ usedFor }) {
+  const [userInput, setUserInput] = useState('');
+  const [userSubmit, setUserSubmit] = useState('');
+
+  const placeholder = () => {
+    switch (usedFor) {
+      case 'nickname':
+        return '닉네임';
+      case 'search':
+        return '검색어';
+      case 'title':
+        return '제목';
+      default:
+        return '내용';
+    }
+  };
```

Comment:
![image](https://user-images.githubusercontent.com/87893624/184498812-ab1491ce-6519-4f2d-8b1c-51a44e5e4d66.png)
[참고 블로그](https://choisuhyeok.tistory.com/58)

현재 조건에 따라 return 하는 것이 다르고, 이것을 switch문을 사용해서 구현했는데 객체를 사용해 lookup table 을 만들면 더 깔끔하게 만들 수 있습니다.

```suggestion
const placeholder = {
  nickname: '닉네임',
  search: '검색어',
  title: '제목',
};

const getPlaceholder = (usedFor) => placeholder2[usedFor] || '내용';

// 함수를 사용하거나 객체를 바로 사용하거나
console.log(placeholder['nickname']); // 닉네임
console.log(getPlaceholder('nickname')); // 닉네임

console.log(getPlaceholder('search')); // 검색어
console.log(getPlaceholder('title')); // 제목
console.log(getPlaceholder()); // 내용
```

위와 같이 만들 수 있겠으나 이 코드는 참고용으로 이해하시고, 태영님한테 맞게끔 추가적으로 손 보면 좋을 것 같습니다. 아래에서 사용한 다른 switch문도 이렇게 객체를 이용한 형태로 수정하면 좋을 것 같아요😊